_people/aydin_saribudak.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "Aydin Saribudak"
+collection: people
+permalink: /people/aydins
+position: phd
+title: "PhD Student"
+joined: 2019
+avatar: aydins.jpg
+contacts:
+  email: "aydin.saribudak@rutgers.edu"
+  office: "CoRE 707"
+  [scholar][scholarlink] 
```

Comment:
This line produces the following error:
```
Error: YAML Exception reading /Users/iparask/radical-group/_people/aydin_saribudak.md: (<unknown>): could not find expected ':' while scanning a simple key at line 12 column 3
                    done in 0.
```

As a result your page does not get created. Check [here](https://raw.githubusercontent.com/radical-group/radical-group.github.io/master/_people/ioannis_par.md) to see how to incorporate a Google scholar link